README.md

```diff
@@ -1,73 +1,214 @@
-#### Response
-Returns all applicants
+###### Response: Returns all applicants
+  ```JSON
+  {
+      "success": true,
+      "data": [
+          {
+              "id": 1,
+              "username": "Anonymous Giraffe",
+              "email": "gaby@hireup.com",
+              "bio": "Noodle's mom!",
+              "skills": [
+                  "rails",
+                  "ruby"
+              ],
+              "values": [
+                  "creativity"
+              ]
+          },
+          {
+              "id": 2,
+              "username": "Famous Hippo",
+              "email": "ruthie@hireup.com",
+              "bio": "Noodle's mom's accountabilabuddy!",
+              "skills": [
+                  "rails",
+                  "flask"
+              ],
+              "values": [
+                  "creativity",
+                  "mentorship"
+              ]
+          }
+      ]
+  }
+  ```
+
+#### GET `/api/v1/applicants/:applicant_id`
+###### Response:
+  ```JSON
+  {
+      "success": true,
+      "data": {
+          "id": 1,
+          "username": "Fancy Chipmunk",
+          "email": "greyson@hireup.com",
+          "bio": "I'm the best one you could possibly hire",
+          "skills": [
+              "rails",
+              "ruby"
+          ],
+          "values": [
+              "creativity"
+          ]
+      }
+  }
+  ```
+
+#### POST `/api/v1/applicants/`
+###### Request: Email, skills, and values are required fields
+  ```JSON
+  {
+      "first_name": "Greyson",
+      "last_name": "Johns",
+      "bio": "I'm the best one you could possibly hire",
+      "email": "greyson@google.com",
+      "username": "Chipmunk",
+      "skills": [2, 3],
+      "values": [2]
+  }
+  ```
+
+###### Response: Creates new applicant and returns attributes including the new user's id
+  ```JSON
+  {
+      "success": true,
+      "data": {
+          "id": 6,
+          "username": "Chipmunk",
+          "email": "greyson@google.com",
+          "bio": "I'm the best one you could possibly hire",
+          "skills": [
+              "flask",
+              "ruby"
+          ],
+          "values": [
+              "mentorship"
+          ],
+          "success": true
+      }
+  }
+  ```
+
+###### Error handling: Request body with empty arrays for either skills or values when being created will produce a 400 error message  
+Example erroneous request:
+  ```JSON
+  {
+      "first_name": "Greyson",
+      "last_name": "Johns",
+      "bio": "I'm the best one you could possibly hire",
+      "email": "greyson@google.com",
+      "username": "Chipmunk",
+      "skills": [2, 3]
+  }
+  # missing "values" array
+  ```
+Error message response:
+  ```JSON
+  {
+      "success": false,
+      "error": 400,
+      "errors": [
+          "required 'values' parameter is missing"
+      ]
+  }
+  ```
+
+<!-- commenting this out until we have this request body figured out -->
+<!-- #### PATCH  `/api/v1/applicants/:applicant_id`
```

Comment:
`ApplicantResource` class seems to indicate we have access to a PATCH endpoint but the request body example below doesn't seem to be complete - I tried this and also specifying the applicant id and a few other formats, but none worked. Just want to confirm whether this functionality is available and make sure we have accurate documentation for it.